src/components/KInputSwitch/KInputSwitch.vue

```diff
@@ -160,13 +160,19 @@ export default {
 
 <style lang="scss" scoped>
 @import '@/styles/variables';
-@import '@/styles/functions';
+@import "@/styles/tmp-variables";
 
-.k-switch,
-.k-input-switch {
+$width: 44px;
+$height: 24px;
+$transition: $tmp-animation-timing-2 linear;
```

Comment:
suggestion: can you name these a little more specifically if only used here? e.g. `$k-input-switch-width` etc.